SwiftWisdom/Core/Foundation/Defaults.swift

```diff
@@ -41,3 +41,14 @@ extension EnumSettingsKeyAccessible {
         return rawValue
     }
 }
+
+extension UserDefaults {
+
+    /// Removes all stored objects from UserDefaults.
+    func ip_clear() {
```

Comment:
Since `UserDefaults` is declared `open` this is available outside of the module by default.